backend/municipal-services/ingestion-service/app/utils/convertor.py

```
@@ -0,0 +1,224 @@
+import json
+from json import JSONDecodeError
```

Ruff (0.8.2)

`json.JSONDecodeError` imported but unused

Remove unused import: `json.JSONDecodeError`

(F401)